sqoop/README.MD

```diff
@@ -0,0 +1,74 @@
+# Sqoop 
+
+This initialization action installs [Sqoop](http://sqoop.apache.org/) on a [Google Cloud Dataproc](https://cloud.google.com/dataproc) cluster. Sqoop is being compiled from trunk branch of official [github](https://github.com/apache/sqoop) mirror.
```

Comment:
Compiling from `trunk` is a little scary. We should at least compile from a release tag and just cherry-pick the necessary commit in.

@pmkc will we be able to backport [the commit](https://github.com/apache/sqoop/commit/e13dd21209c26316d43350a23f5d533321b61352) into Dataproc so we don't need to compile Sqoop in this init action?

Reply:
This CL should not be compiling at all. It should certainly ignore BigTable until Sqoop is fixed in Bigtop with that patch

Reply:
Let me know when Sqoop will be available in Bigtop. I will test it and update this PR properly, so far I will just update docs.